src/tasks/daily/daily_battle_mixin.py

```
@@ -399,41 +431,78 @@ def _gather_retry_navigate(self, stage_name, category_name):
         self._navigate_via_zip_line(stage_name)
         #
         self.navigate_until_target(target_ocr_pattern=re.compile("领取"), nav_feature_name=fL.gather_icon_out_map, time_out=60)
-        result = self.wait_ocr(match=re.compile("领取"), box=self.box.bottom_right, time_out=5)
+        click_key = "放弃" if is_extra_mode else "领取"
+        result = self.wait_ocr(match=re.compile(click_key), box=self.box.bottom_right, time_out=5)
         if not result:
-            self.log_info("二次寻路失败：没有找到『领取奖励』按钮")
+            self.log_info(f"二次寻路失败：没有找到『{click_key}』按钮")
```

Ruff (0.15.12)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)